src/main/java/module-info.java

```diff
@@ -17,6 +17,9 @@
  */
 
 module org.jboss.modules {
+  // for consumers of this module
+  requires java.se;
```

Comment:
When we require java.se we can drop all of the following redundant requirements:
 *  requires java.desktop;
 * requires java.instrument;
 * requires java.logging;
 * requires java.management;
 * requires java.prefs;

Reply:
Good point.